app/lib/shared/analyzer_client.dart

```diff
@@ -65,4 +86,43 @@ class AnalysisView {
     list.sort();
     return list;
   }
+
+  /// Returns the raw health score between -1.0 and 1.0
+  /// These need to be normalized for search and frontend purposes.
+  double get health {
```

Comment:
Since this is a computation on the immutable analysis result, why are we not caching it somewhere? Seems like we should avoid re-computing this.